### DISCUSSION TITLE
Add an adjustable cursor trail update rate slider

### DISCUSSION THREAD

**Discussion by Siiiuuuuu** [ppy/osu#30975] [Ideas]
It seems that the cursor trail update rate is locked to 60hz. It would be nice to have an option to make this adjustable to each users liking. Sometimes when you alt-tab out of osu and come back into the game, the cursor goes ultra smooth. When you start to play however, it goes back to normal again.

Example of this: 

https://github.com/user-attachments/assets/2760ad15-8aea-4940-9288-a0d0a5fe423f

I've also seen this being asked by other users before, so I'm sure this would be a welcomed change. :) 

Preferably, this would be a slider, starting from 1 to the users monitor refresh rate. A small dropdown window (next to the slider or wherever) for values like 30, 60, 120, 144, 240, 360, would also make a great workflow boost.

**Comment by Siiiuuuuu** [comment]
bumping this since this might be nice to have bundled in with the new song select
(note: the video above is only showing the cursor running at 165hz, it doesnt transition here)

**Reply by Siiiuuuuu** [reply]
here is a version where i switch between normal (60hz) and smooth (165hz) by alt tabbing 

https://github.com/user-attachments/assets/e34dd62e-bdbf-4817-b993-adcd752697e1

**Reply by ghost** [reply]
If it wouldn't cause performance issues, I'd prefer the dropdown options allow it to update faster than the monitor refresh rate. I think it wouldn't effect my playing/reading if two dots appeared in one monitor frame, and would just make it feel more smooth, but I'm not sure as I haven't tried it.
